app/views/devise/registrations/forms/_new.html.erb

```diff
@@ -16,54 +16,26 @@
 
   <div class='row row-fields'>
     <div id='email' class='col-sm-8 col-sm-offset-2'>
-      <div class='form-group'>
```

Comment:
FYI – this is what this page looks like now:

<img width="715" alt="screen shot 2016-09-04 at 9 24 36 am" src="https://cloud.githubusercontent.com/assets/572804/18232294/8261d100-7281-11e6-8216-48630161ed4b.png">